### PR TITLE
Feature/list all blogs

### PR DESCRIPTION
**Pull request by Rnf-u**
yeeeeey mana, finally

### CODE REVIEW

File: yuma/posts/views.py [hunk 1]
```diff
@@ -10,15 +10,5 @@ class PostsViewSet(viewsets.ViewSet):
     renderer_classes = [TemplateHTMLRenderer]
 
     def list(self, request):
-        """List Posts
-
-        Lists all posts.
-        """
+        #returns all post records
```
Comment (Cymmer) [inline yuma/posts/views.py:13]: Kung mag document kag endpoint, 
```
"""Name Of Endpoint

What the Endpoint does
"""
```

**Review by Cymmer — CHANGES_REQUESTED** [review]
Add ug space between `{{ varname }}`

File: yuma/posts/views.py (hunk 1 above)
Comment (Rnf-u) [reply yuma/posts/views.py:13]: ahh gege noted bos

File: yuma/posts/templates/index.html [hunk 2]
```diff
@@ -76,26 +76,23 @@ <h2>All Posts</h2>
...
           <div class="Blogs__card__left">
-            <img style="width:100%; height: 100%" src="{{ post.thumbnail }}" />
+            <img src="{{ post.thumbnail }}" />
           </div>
           <div class="Blogs__card__right">
             <article class="Blogs__card__right__content">
               <div class="Blogs__card__right__tags">
-                {% for category in post.categories.all %}
-                <span class="Category">{{ category.name }}</span>
+                {% for c in post.categories.all %}
```
Comment (Cymmer) [inline yuma/posts/templates/index.html:84]: ayaw pag `c` oy. `category` gamita kay para masabtan dayon

**Review by Cymmer — CHANGES_REQUESTED** [review]

File: yuma/posts/templates/index.html (hunk 2 above)
Comment (Rnf-u) [reply yuma/posts/templates/index.html:84]: Ayy HAHAHAHA abi mn nako ganina mao nay rason ngano di mugana, ang categories mn diay HAHAAHHA gegegeg noted

**Pull request merged**